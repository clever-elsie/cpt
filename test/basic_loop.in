let n=15;

let sumof=\sum_{i=1}^{n} i;

let prodof=\prod_{i=2}^{n} i;

\print(n,sumof,prodof);