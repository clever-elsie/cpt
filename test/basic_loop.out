15
120
1307674368000
3
